# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gfnrename/hotfix-autogen.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gfnrename-0.6.orig/autogen.sh	2007-06-16 09:02:34.000000000 +0200
+++ gfnrename-0.6/autogen.sh	2026-07-26 06:34:50.535025418 +0200
@@ -119,8 +119,8 @@
 
 if test x$NOCONFIGURE = x; then
   echo Running $srcdir/configure $conf_flags "$@" ...
-  $srcdir/configure $conf_flags "$@" \
-  && echo Now type \`make\' to compile $PKG_NAME
+  $srcdir/configure $conf_flags "$@" && \
+  echo Now type \`make\' to compile $PKG_NAME
 else
   echo Skipping configure process.
 fi
